pkg/ebpf/events_pipeline.go

```diff
@@ -363,7 +369,10 @@ func (t *Tracee) sinkEvents(ctx context.Context, in <-chan *trace.Event) <-chan
 
 			select {
 			case t.config.ChanEvents <- *event:
-				t.stats.EventCount.Increment()
+				err := t.stats.EventCount.Increment()
+				if err != nil {
+					logger.Error("Incrementing events count", "error", err)
```

Comment:
So I did a quick check and running at ~15k events/sec (which is a lot), it would take us ~19498080 years to overflow this, which can be the only error iirc.
So considering the hotpath, i'm wondering if there should:
1. Even be an error here
2. If we keep 1, an error check for a log

And considering this case, which is our quickest rising counter besides `FilteredEvents`, I would say it's even more irrelevant in other counters.

Reply:
@NDStrahilevitz thanks for digging into this. Besides Increment() and Decrement(), Counter has `Set()`, which was implemented during the process of the aggregated bpf logs PR, in which it would be used just to set the count instead of calculating it per cpu.

`Set()` could easily make a value near to a possible wrap around, but that function is not being used.

Another code that can circumvent straight Increment is `t.stats.BPFLogsCount.Increment(uint64(bpfLog.Count()))` or others that make the Increment using runtime values.

I agree with you that odds are low here and considering the hot paths I would change them to `_ = *.Increment()/Decrement()`, making explicit that it can err and for the  BPFLogsCount I must keep the checking, since received values are out of direct control.

Reply:
Done. 👍🏼